pkg/test/framework/components/echo/kube/deployment.go

```diff
@@ -89,13 +89,19 @@ spec:
 `
 
 	deploymentYAML = `
+{{- $revisions := .Revisions }}
 {{- $subsets := .Subsets }}
 {{- $cluster := .Cluster }}
 {{- range $i, $subset := $subsets }}
+{{- range $j, $revision := $revisions }}
 apiVersion: apps/v1
 kind: Deployment
 metadata:
+{{- if $.IsCrossVersion }}
```

Comment:
what does `IsCrossVersion` mean? That there are multiple versions configured?

Reply:
Yes, it just means that `--istio.test.versions` was set. Maybe `IsMultiVersion` would be a better name here...

Reply:
+1 for `IsMultiVersion`